sdk/resourcemanager/api-specs.json

```diff
@@ -106,7 +106,7 @@
     "dir": "azure-resourcemanager-cosmos",
     "source": "specification/cosmos-db/resource-manager/readme.md",
     "package": "com.azure.resourcemanager.cosmos",
-    "args": "--payload-flattening-threshold=1 --tag=package-2019-08-preview"
+    "args": "--payload-flattening-threshold=0a --tag=package-2019-08-preview"
```

Comment:
what does `0a` mean here?

Reply:
Typo, should be `0`, fixed. Current m4 got bug on flatten of type which inherits parent type but not having any property. So do not flatten cosmos (which is affected by the bug).